app/assets/stylesheets/custom/editor.less

```diff
@@ -0,0 +1,61 @@
+/*******************************
+     .editor
+*******************************/
+
+
+.editor {
+
+  &__sidebar__column:not(:last-child) {
+    border-right: 1px solid @darkGray;
+    padding: 0 1em;
+  }
+
+  &__sidebar {
+    background: white;
+    padding: 0.5em;
+  }
+
+  &__sidebarwrapper {
+    background-color: @veryLightGray;
+    border-radius: 15px 0 0 15px; // starts upper left corner
+  }
+
+  &__sidebarmenu {
+    background-color: @darkGray;
+
+    &__item {
+      padding: 0.25em;
+      border-bottom: 1px solid @veryLightGray;
+    }
+
+    &__icon {
+      margin: 0em !important;
+    }
+
+    &__button {
+      margin: 1em 0.75em;
+      cursor: pointer;
+      display: inline-block;
+      min-height: 1em;
+      outline: none;
+      border: none;
+      vertical-align: baseline;
+      background: #E0E1E2 none;
+      color: rgba(0, 0, 0, 0.6);
+      padding: 0.78571429em 1em 0.78571429em;
+      text-transform: none;
+      text-shadow: none;
+      font-weight: bold;
+      line-height: 1em;
+      font-style: normal;
+      text-align: center;
+      text-decoration: none;
+      border-radius: 0.28571429rem;
+      box-shadow: 0px 0px 0px 1px transparent inset, 0px 0em 0px 0px rgba(34, 36, 38, 0.15) inset;
+      user-select: none;
+      transition: opacity 0.1s ease, background-color 0.1s ease, color 0.1s ease, box-shadow 0.1s ease, background 0.1s ease;
+      will-change: '';
+      -webkit-tap-highlight-color: transparent;
```

Comment:
Whoa, overkill? If you actually need all of these properties you are probably doing something wrong 😅